game/animvalue.hh

```diff
@@ -69,8 +71,8 @@ class AnimAcceleration {
 			m_velocity = 0.0;
 			return m_position = m_target;
 		}
-		std::size_t rounds = 1.0 + 1000.0 * duration; // 1 ms or shorter timesteps
-		double t = duration / rounds;
+		std::size_t rounds = static_cast<size_t>(1.0 + 1000.0 * duration); // 1 ms or shorter timesteps
+		double t = duration / static_cast<double>(rounds);
```

Comment:
Wait... Double? I thought we were converting to float?

Reply:
The idea is to use double for frequencies and also stuff involving time (such as animations). I've also in some places preserved the double during calculation and then cast to float to actually use the results.